applications/StructuralMechanicsApplication/custom_constitutive/elastic_isotropic_3d.cpp

```diff
@@ -171,6 +171,38 @@ void ElasticIsotropic3D::FinalizeMaterialResponseCauchy(ConstitutiveLaw::Paramet
 /***********************************************************************************/
 /***********************************************************************************/
 
+bool ElasticIsotropic3D::Has(const Variable<bool>& rThisVariable)
+{
+    if (rThisVariable == REQUIRES_MATERIAL_INITIALIZATION) {
+        return true;
+    } else if (rThisVariable == REQUIRES_MATERIAL_FINALIZATION) {
+        return true;
+    } else {
+        return BaseType::Has(rThisVariable);
+    }
+
+    return false;
+}
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+bool& ElasticIsotropic3D::GetValue(const Variable<bool>& rThisVariable, bool& rValue)
+{
+    if (rThisVariable == REQUIRES_MATERIAL_INITIALIZATION) {
+        rValue = false;
```

Comment:
both `false` ?? 
if yes can they be combined into single condition ?

Reply:
This was a dirty solution, i would vote for using local flags instead